src/Components/Components/src/CascadingParameterState.cs

```diff
@@ -24,6 +24,7 @@ public CascadingParameterState(string localValueName, ICascadingValueComponent v
             ValueSupplier = valueSupplier;
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2072:RequiresUnreferencedCode", Justification = "Requires a gesture that ensures components are always preserved. https://github.com/mono/linker/issues/1806")]
```

Comment:
I wonder if it would be better to baseline these in an .xml file, like we do in dotnet/runtime. That way we know to come back to them when the linker feature is done.

The benefit is that the application will still get a warning, because what this is doing is unsafe.